vlib/v/checker/tests/comptime_dump_fields_var_test.vv

```diff
@@ -0,0 +1,21 @@
+struct Foo {
+	a string
+	b ?string
+	c int
+	d ?int
+	e struct {
+		i ?int
+	}
+}
+
+fn sei_oq_la[T](val T) {
```

Comment:
what does `sei_oq_la` mean?

Reply:
oh, it is directly from the issue 🤔

Reply:
It's a portuguese phrase for "whatever". I was just copying from the original test case's issue.